### PR TITLE
Use 2 partitions in planner tests

### PR DESCRIPTION
**Pull request by andygrove**
Use 2 partitions so that we have distributed plans with multiple query stages

### CODE REVIEW

File: src/planner.rs [hunk 1]
```diff
@@ -394,8 +394,8 @@ mod test {
 
         let file = format!("testdata/queries/q{n}.sql");
         let sql = fs::read_to_string(&file)?;
-        let config = SessionConfig::new().with_target_partitions(1);
-        let ctx = SessionContext::with_config(config);
+        let config = SessionConfig::new().with_target_partitions(2);
```
Comment (andygrove) [inline src/planner.rs:397]: This is the main change

File: README.md [hunk 2]
```diff
@@ -145,26 +145,6 @@ export TPCH_DATA_PATH=`pwd`/data
 cargo test
 ```
 
-Tests compare plans with expected plans, which unfortunately contain the
```
Comment (edmondop) [inline README.md:148]: If this is not needed anymore, can we also remove the script ?

Comment (andygrove) [reply README.md:148]: It was removed in my previous PR, but I missed this documentation

Comment (edmondop) [reply README.md:148]: Ooops, my previous review was not deep enough

**Review by edmondop — APPROVED** [review]

**Pull request merged**